packages/hooks/src/components/use-form/use-form.ts

```
@@ -150,30 +154,77 @@ const useForm = <T extends ObjectType>(props: UseFormProps<T>) => {
+          return Array.from(validateField).map((validate) =>
+            validate(key, deepGet(context.value, key), context.value, config),
+          );
         });
-    });
+
+        let validatorValue = context.value;
+        if (fields) {
+          const fieldArray = isArray(fields) ? fields : [fields];
+          validatorValue = fieldArray.reduce(
+            (prev, cur) => ({
+              ...prev,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)